### ISSUE TITLE
Please move FindSuperCollider3.cmake into the SC project

### ISSUE THREAD

**Issue by yurivict** [supercollider/sc3-plugins#171]
It would be more logical for FindSuperCollider3.cmake to reside in SC project, not sc3-plugins project.

Since you own both, this should be easy to do.

**Comment by brianlheim** [comment]
Hey @yurivict !

> It would be more logical for FindSuperCollider3.cmake to reside in SC project, not sc3-plugins project.

I don't agree, can you please explain?

> Since you own both, this should be easy to do.

Yes, but it would create issues with versioning. I think you are going to have to provide more of a rationale to justify that.

**Comment by yurivict** [comment]
> I don't agree, can you please explain?

There is one SuperCollider, and potentially many other cmake projects that need to use it. This is logical if SC provided the .cmake file, instead of every dependency copying it.

For example:
* the package phonon-4.9.1_1 provides its .cmake file: /usr/local/lib/cmake/phonon/PhononConfig.cmake
* the package libftdi1-1.3_3 provides /usr/local/lib/cmake/libftdi1/UseLibFTDI1.cmake
* Same principle works when projects provide their .pc files too.

**Comment by brianlheim** [comment]
@yurivict thanks! That makes sense. To be clear, are you asking for the file to also be removed from this repository? And would that cause issues if sc3-plugins is built with a supercollider source tree using an older branch/commit that doesn't include the cmake module?

**Comment by yurivict** [comment]
@brianlheim You should first create it in the SC tree, and after it gets released this way delete it from sc3-plugins.

**Comment by brianlheim** [comment]
@yurivict 

> And would that cause issues if sc3-plugins is built with a supercollider source tree using an older branch/commit that doesn't include the cmake module?

It would, right?

The issue is that it's very likely people will need to build a current version of sc3-plugins using an old version of the API headers (pre 3.9.0). Meaning that it's very likely people will need to build under the condition that the cmake module doesn't exist in either repository.

What do you think would be a good solution here?

**Comment by yurivict** [comment]
> It would, right?

Right, so you can keep both for a while, until older branches become irrelevant.

**Comment by LFSaw** [comment]
curious: What's the point of a `cmake` file within the host repo (supercollider) that defines `SC_FOUND`? this would always be true if the repo (and thus the cmake file)  is there and otherwise the call to `FindSuperCollider` would result in an error, right?

**Comment by yurivict** [comment]
```SC_FOUND``` shouldn't be there. Can it build without SC?
